source/MissionPanel.cpp

```diff
@@ -649,13 +687,42 @@ Point MissionPanel::DrawPanel(Point pos, const string &label, int entries) const
 		edgePos.Y() -= dy;
 	}
 
+
 	const Font &font = FontSet::Get(14);
 	pos += Point(10., 10. + (20. - font.Height()) * .5);
-	font.Draw(label, pos, selected);
+
+	// Panel sorting
+	const Color &text= *GameData::Colors().Get("medium");
```

Comment:
It's a small thing, but we could be initiating this on line 663, and then get the separatorLine based on it (also because its not a coincidence they both use medium sized colour), also that way all colours are defined at the start of the function.